src/SeqViz/handlers/selection.tsx

```diff
@@ -409,21 +423,20 @@ const withSelectionHandler = WrappedComp =>
      * Update the selection in state. Only update the specified
      * properties of the selection that should be updated.
      */
-    setSelection = newSelection => {
+    setSelection = (newSelection: SeqVizSelection) => {
       // @ts-expect-error ts-migrate(2339) FIXME: Property 'setSelection' does not exist on type 'Re... Remove this comment to see the full error message
       const { setSelection } = this.props;
-
       if (
         // @ts-expect-error ts-migrate(2339) FIXME: Property 'selection' does not exist on type 'Reado... Remove this comment to see the full error message
         newSelection.start === this.props.selection.start &&
         // @ts-expect-error ts-migrate(2339) FIXME: Property 'selection' does not exist on type 'Reado... Remove this comment to see the full error message
         newSelection.end === this.props.selection.end &&
         // @ts-expect-error ts-migrate(2339) FIXME: Property 'selection' does not exist on type 'Reado... Remove this comment to see the full error message
-        newSelection.ref === this.props.selection.ref
+        newSelection.ref === this.props.selection.ref &&
+        newSelection.type !== "ANNOTATION" // to support reclicking the annotation and causing it to fire a la gh issue https://github.com/Lattice-Automation/seqviz/issues/142
```

Comment:
Imo if we're changing this for one type we should change for all of them. The bug report could have applied to translations, finds, etc too